jobs/integr8ly/poc-upgrade.yaml

```diff
@@ -0,0 +1,95 @@
+---
+
+- job:
+    name: poc-upgrade
+    project-type: pipeline
+    description: "Installs Integreatly remotely from Jenkins slave to QE Pony cluster."
+    sandbox: false
+    concurrent: true
+    parameters:
+        - string:
+            name: REPOSITORY
+            default: https://github.com/integr8ly/installation.git
+            description: "Repository of the Integreatly installer"
+        - string:
+            name: BRANCH
+            default: 'master'
+            description: "Branch of the installer repository"
+        - string:
+            name: CLUSTER_URL
```

Comment:
I don't see cluster_url used anywhere apart from `verify input` stage. Is it needed?